tasks/install_shell.sh

```diff
@@ -417,24 +435,24 @@ case $platform in
   "SLES")
     info "SLES platform! Lets get you an RPM..."
     gpg_key="${tmp_dir}/RPM-GPG-KEY-puppet"
-    do_download "https://yum.puppetlabs.com/RPM-GPG-KEY-puppet" "$gpg_key"
+    do_download "https://yum.puppet.com/RPM-GPG-KEY-puppet" "$gpg_key"
```

Comment:
not ${yum_source}?

Reply:
Right now those keys only exist on the public downloads servers, which means you can't install from nightlies. For now this needs to stay a full URL until https://tickets.puppetlabs.com/browse/RE-12161 is resolved.